app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -154,13 +157,14 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                     }
                     Timber.d("Success! %s", attendees.toList().toString())
                 }, {
-                    if (it.message.equals(TICKET_CONFLICT_MESSAGE)) {
-                        ticketSoldOut.value = true
-                    } else {
-                        message.value = "Unable to create Attendee!"
-                        Timber.d(it, "Failed")
-                        ticketSoldOut.value = false
-                    }
+                    if (createAttendeeIterations + 1 == totalAttendee)
+                        if (it.message.equals(TICKET_CONFLICT_MESSAGE)) {
```

Comment:
these two if statements can be combined

Reply:
then we will have to use else if so no point

Reply:
`if (createAttendeeIterations + 1 == totalAttendee && it.message.equals(TICKET_CONFLICT_MESSAGE)){}`

`else{}`

Reply:
then we will have to use else if because if not multiple toast and logs will be created

Reply:
Okay